scriptworker/constants.py

```diff
@@ -329,10 +329,14 @@
                 'project:mobile:focus:googleplay:product:focus': 'focus-repo',
                 'project:mobile:focus:releng:signing:cert:release-signing': 'focus-repo',
 
-                'project:mobile:reference-browser:releng:signing:cert:release-signing': 'reference-browser-repo',
                 'project:mobile:reference-browser:releng:googleplay:product:reference-browser': 'reference-browser-repo',
+                'project:mobile:reference-browser:releng:signing:cert:release-signing': 'reference-browser-repo',
+
+                'project:mobile:firefox-tv:releng:googleplay:product:firefox-tv': 'firefox-tv-repo',
+                'project:mobile:firefox-tv:releng:signing:cert:production-signing': 'firefox-tv-repo',
             }),
             'application-services': frozendict({
+                'project:releng:signing:cert:release-signing': 'application-services-repo',
```

Comment:
oops.

Reply:
Oops, my stash-and-apply must've goof'd.
I should've checked this PR diff right after creating the PR, sorry about this!